mutiny-core/src/storage.rs

```diff
@@ -268,6 +273,28 @@ pub trait MutinyStorage: Clone + Sized + Send + Sync + 'static {
         self.get_data(MNEMONIC_KEY)
     }
 
+    /// Insert a nsec into the storage
+    async fn insert_nsec(&self, nsec: SecretKey) -> Result<(), MutinyError> {
```

Comment:
Hard NACK. Nsec should not be pasted into a browser site.

Reply:
Should discuss IRL but paul and I's thinking was we already doing worse with a bitcoin seed

Reply:
Huge difference in a mutiny seed being used for mutiny things and importing another seed/private key from somewhere else for other purposes. 

Find another way, there's others that exist. Maybe ask Pablo...

Reply:
There's a few things I think we can do here but I would absolutely not allow nsec pasting here. 

1. Nostr plugins for web browsers that already have them
2. Create an nsec from the derivation path and use that for users that don't already have nostr
  2a. Some delegated nostr stuff so you can use the mutiny derivation seed as if it were the user's main key (don't think this really exists yet)
3. nsec bunker for everyone else